ods_ci/tests/Tests/0700__feature_store/test-run-feast-operator-e2e.robot

```diff
@@ -22,3 +22,10 @@ Run runTestWithRemoteRegistryFunction test
     ...     FeatureStore
     ...     RHOAIENG-14799
     Run Feast Operator E2E Test    TestRemoteRegistryFeastCR
+
+Run runTestApplyAndMaterializeFeastDefinitions test
```

Comment:
Will we need new release in upstream to get this test in test suite ?

Reply:
Since RHAOI 2.21 as `v0.49.0` feast changes , not required for now.  I did cherry pick of [commit](https://github.com/feast-dev/feast/pull/5389/commits/30952964e753468489d7fbf7def82fe7ac95a499) against `v0.49.0` release branch to custom branch `adjustment-release-v0.49.0`for now.